src/main/java/com/github/devotedmc/hiddenore/listeners/WorldGenerationListener.java

```diff
@@ -16,41 +15,37 @@
 import org.bukkit.event.world.ChunkPopulateEvent;
 import org.bukkit.inventory.ItemStack;
 
-import com.github.devotedmc.hiddenore.BlockConfig;
-import com.github.devotedmc.hiddenore.Config;
-import com.github.devotedmc.hiddenore.HiddenOre;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.logging.Level;
 
 /**
- * Populator to strip out blocks selectively from a world during generation. 
- * 
+ * Populator to strip out blocks selectively from a world during generation.
+ *
  * @author ProgrammerDan
  */
 public class WorldGenerationListener implements Listener {
 
 	Set<Material> toReplace = null;
 	Material replaceWith = null;
 	String worldName = null;
-	
+
 	/**
-	 * When creating, pass in a config with three sub-elements
+	 * When creating, pass in a config with two sub-elements
 	 * <br/>
 	 * <code>
-	 *   world: world_name
 	 *   replace:
 	 *   - IRON_ORE
 	 *   - REDSTONE_ORE
 	 *   with: STONE
 	 * </code>
 	 * <br/>
 	 * This should be specified per world.
-	 * 
+	 *
 	 * @param config The world-specific config.
-	 * 
+	 *
 	 */
 	public WorldGenerationListener(ConfigurationSection config) {
-		if (config.contains("world")) {
-			worldName = config.getString("world");
```

Comment:
Just wanted to highlight that worldName _must_ be defined somewhere or this whole shebang won't work.

Look further down -- if worldName is null, the listener just drops out.

I'd recommend just adding a new param here to the constructor -- `String worldName` -- then setting it directly. Least effort change -- plus where you build these WorldGenerationListeners above _has_ the worldName handy, so it works out.